example/70-workloadidentity-provider.yaml

```
@@ -0,0 +1,16 @@
+# WorkloadIdentity containing provider-specific details for account that shall be used to deploy the shoot into.
+---
+apiVersion: security.gardener.cloud/v1alpha1
+kind: WorkloadIdentity
+metadata:
+  name: my-provider-account
+  namespace: garden-dev
+spec:
+  audiences:
+  - gardener
+  targetSystem:
+    type: <some-provider-name> # {aws,azure,gcp,...}
+    providerConfig:
+      apiVersion: <some-provider-name>.security.gardener.cloud/v1alpha1
```

```suggestion
      apiVersion: <some-provider-name>.extensions.gardener.cloud/v1alpha1
```

We probably want to avoid the burden of introducing a new, dedicated API group just for this. Let's reuse the existing API groups of the extensions?